ncm-spma/src/main/pan/components/spma/yum/light.pan

```diff
@@ -2,8 +2,10 @@
 # ${developer-info}
 # ${author-info}
 
-@{spma backend specific statements for spmalight}
+@{spma yum backend specific statements for spmalight}
```

Comment:
I think I missed something - where do we describe the purpose of these "light" variants?

Reply:
https://github.com/quattor/configuration-modules-core/blob/master/ncm-spma/src/main/pan/components/spma/light.pan

Reply:
👍  - this issue came up just this week, I didn't realise this was designed to fix!